app/utils.py

```diff
@@ -112,3 +113,26 @@ def email_address_is_nhs(email_address):
     return email_address.lower().endswith((
         '@nhs.uk', '@nhs.net', '.nhs.uk', '.nhs.net',
     ))
+
+
+def get_notification_table_to_use(service, notification_type, process_day, has_delete_task_run=False):
+    """
+    Work out what table will contain notification data for a service by looking up their data retention.
+
+    Make sure that when you run this you think about whether the delete task has run for that day! If it's run, the
+    notifications from that day will have moved to NotificationHistory. By default we assume it hasn't run, since
```

Comment:
since what?

Reply:
i had a think and the default value felt dangerous (as in, easy to ignore and forget about). so i've just made it non-optional and changed this comment.